README.md

```diff
@@ -27,15 +29,23 @@ sdk.dir=/Applications/Android Studio.app/sdk
 Add your simperium appid and key to Simplenote/gradle.properties, and an empty googleAnalyticsId:
 
 ```
-simperiumAppId=SIMPERIUM_APP_ID
-simperiumAppKey=SIMPERIUM_KEY
+simperiumAppId=yield-cannons-00f
+simperiumAppKey=f4d9e4bab5fd40909481d91ae1f0d932
```

Comment:
@dmsnell @roundhill Sorry about that. We forgot to remove it since we have not used readme.md much